msbuild/Xamarin.iOS.Tasks.Core/Xamarin.iOS.Common.targets

```diff
@@ -1692,7 +1692,7 @@ Copyright (C) 2013-2016 Xamarin. All rights reserved.
 			Keychain="%(_AppExtensionCodesignProperties.Keychain)"
 			Entitlements="%(_AppExtensionCodesignProperties.Entitlements)"
 			ResourceRules="%(_AppExtensionCodesignProperties.ResourceRules)"
-			Resource="$(_AppBundlePath)PlugIns\%(_AppExtensionCodesignProperties.Identity)"
+			Resources="$(_AppBundlePath)PlugIns\%(_AppExtensionCodesignProperties.Identity)"
```

Comment:
`Identity` is just the path. This location needs `Identity` because it is constructing a new path to be used by the codesign task.